releasenotes/v0.10.0.md

```diff
@@ -0,0 +1,115 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v0.9.0
+
+## :warning: Breaking Changes
+
+- Bump Kubernetes api, controller-runtime and decouple webhooks from api (#2390)
+- Only accept HTTPHeadersRef in same namespace
```

Comment:
PR link missing

Reply:
There are no PR. But advisory link is needed.